SMLHelper/FishFramework/CustomFish.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace SMLHelper.V2.FishFramework
+{
+    using System;
+    using System.Collections.Generic;
+    using UnityEngine;
+
+    public class CustomFish
+    {
```

Comment:
What is the difference between this and `CustomFishPrefab`, and why would one be used over the other?

Reply:
I created this class to act as a wrapper for CustomFishPrefab that automatically supplies a TechType and registers the prefab, but if you would prefer to have the prefabs created in the same was as other ModPrefab classes thats fine

Reply:
I think I understand now. So effectively this provides two different ways to register the fish.  

Either through `CustomFishPrefab` which inherits from `ModPrefab`    
or instead from `CustomFish` which is provided to the `RegisterFish` method of the `CustomFishHandler` class.

Having multiple options can be helpful in simplifying the code someone else will need to write to integrate into this API.  
Just make sure things don't get too confusing.  
Since everything is strongly typed, there is no issue with someone incorrectly using the APIs here.  
Right now I don't have any strong opinions on which way this should go.